README.md

```diff
@@ -1,69 +1,31 @@
+[![Feature Upvote](https://img.shields.io/static/v1?style=for-the-badge&label=Feature+Upvote&message=Request&color=important&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAAAIKSURBVDhPnZM9aFNRFIDPeWkFlw5uEW3TqojiYqIO71GHSPOGdlNxEgRBEXGTIhkK/g6hKHQQzFYquOggRfFlKAjmKcLLoqIOXRQMqDh0kSrN9Tt58RkERXvg45577/m9Pyq/SSkIdzjnTqEeglHowDI0RLTeiqN36JlkAXAcwPES6nkYgJfwBsxmD+yEVbjIUo1AFjgNgHMO5zuoR+Ghqk4nzeiV7f0UbErYXEc9CLdU9EwSRy5nm/mt26oM5+AGMU8Q/WPRDz3WZ8HfvGXscRI3PuSHt9/GZgROQrv9fjlRIueJbD0+JfMEmTvmLOLqPUOTGhkvWMaiXxlk/gwK2I96OB9nspHM1a5zUDHnm6yZs+sx7cRdYZRW3PjOMAOb8D2CsZSBk3XP9wehYj7H/DTMw2uI4T5UqYwDRFS5EVmBsgUowFsiuzWRcfSzQK9qFdhJf6PUY4wPSDJDy7tbzciqsLbHLEAm6iTGeErVs4MkXiq0tkrWw+xNdtyaVZWJciiPGHdBwarorvaEvRcMn1i3NjMp+eEgZ/IZddEqWIJhYh2wzX8RslQYhmDJ43oWUL6yfG1fYNf3dykG4QZsL6N+oaW7HnfbZnIVyh0ns5Td/z/uwWKqps9dXPd97AW79pX0KfuVHGX931P2eMpPIreuz6QqtaTZ+PWZ+oVAf/zOVFansr7vLPIDh9/cyYgVQMYAAAAASUVORK5CYII=)](https://retroarcher.github.io/featurereq/)
+
+#### Donate
+[![GitHub Sponsors](https://img.shields.io/github/sponsors/ReenigneArcher?label=Github%20Sponsors&style=for-the-badge&logo=githubsponsors)](https://github.com/sponsors/ReenigneArcher)
+[![MEE6](https://img.shields.io/static/v1?style=for-the-badge&label=MEE6&message=Donate&color=success&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAAklEQVR4AewaftIAAAG6SURBVJXBPWgTARiA4fe7H/WKCGoo0qAcIhUJHCSnoIgZYiq4ugk6BEuEimRxk04SwcEhU0pMDY5xcnSK0N22XCexYHSxkpBCB69gc58ntHCcEerzCCmVIMyjVEFLCi4xgT5ID6HV8Zw1EoR9D4LQMQwav8Y6DwiTqW1KO4qoLXtOSMwkNr8ROmON3kfKHUD4N4kUX9Eblx8tdleb9T2D2Fi1AVLk0KQ4Vm0Qk0oQ5lX1IyD8HxUR31DVKiDsU43YHQ1I290eolFEgqhq1RAokbDx+iXd8nl2vm1y4OfgO2/nLrC+9JwkgZKl4JIwc/Um4WCLqeksB46dzJC7/5izxdskKbgWKZlcgUyuQJJh2fi1Z0xiCfQVZkkJhz/Y+fqZP+zjJzh10SNNoG8p9IBZUkafAjaf3OW0bTO6doviizekKfQsEWmp6kNASMhen6O89I6Z7QEfrpSJ+IuKSMtcb9a38gtPs4BPyvDMOb64l9AjR0kTod3xpl4ZxEyRGugKh6YrpkiNmElstVnf8xcWu5Yp05FSAITJ1DalLci9Zc8JiQkplSDMo1RBSwouMYE+SA+h1fGcNRJ+A8xgoQAar6IXAAAAAElFTkSuQmCC)](https://mee6.xyz/m/804382334370578482)
+[![Patreon](https://img.shields.io/static/v1?style=for-the-badge&label=Patreon&message=Donate&color=success&logo=patreon)](https://www.patreon.com/RetroArcher)
+[![Paypal](https://img.shields.io/static/v1?style=for-the-badge&label=PayPal&message=Donate&color=success&logo=paypal)](https://www.paypal.com/paypalme/ReenigneArcher)
+
+#### Support
+[![Discord](https://img.shields.io/discord/804382334370578482?label=Discord&style=for-the-badge&logo=discord)](https://retroarcher.github.io/discord)
+[![Subreddit subscribers](https://img.shields.io/reddit/subreddit-subscribers/retroarcher?label=Reddit&style=for-the-badge&logo=reddit)](https://www.reddit.com/r/RetroArcher)
```

Comment:
`[![Reddit](https://img.shields.io/reddit/subreddit-subscribers/retroarcher?label=Reddit&style=for-the-badge&logo=reddit)](https://www.reddit.com/r/RetroArcher)`